django/urls/resolvers.py

```diff
@@ -405,7 +405,15 @@ def _check_custom_error_handlers(self):
         # All handlers take (request, exception) arguments except handler500
         # which takes (request).
         for status_code, num_parameters in [(400, 2), (403, 2), (404, 2), (500, 1)]:
-            handler, param_dict = self.resolve_error_handler(status_code)
+            try:
+                handler, param_dict = self.resolve_error_handler(status_code)
+            except (ModuleNotFoundError, ViewDoesNotExist) as e:
+                msg = (
+                    "The custom handler{status_code} view could not be imported."
+
+                ).format(status_code=status_code)
+                messages.append(Error(msg, hint=e.args[0], id='urls.E008'))
```

Comment:
I'm not sure that `e.args[0]` is the best way to get the message. `e.msg` works for `ModuleNotFound` but not `ViewDoesNotExist`.

Reply:
If you want the message of the exception, just use `str(e)`?